content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -0,0 +1,209 @@
+
+Prior to uploading a document, there's the option to designate whether the document is a
+:guilabel:`File` or :guilabel:`URL` from the :guilabel:`Type` drop-down field menu.
+
+.. image:: pdf_quote_builder/document-form-uploaded-pdf.png
+   :align: center
+   :alt: A standard document form with an uploaded pdf in Odoo Sales.
+
+.. note::
+    If a PDF is uploaded, the :guilabel:`Type` field is auto-populated to :guilabel:`File`, and it
+    cannot be modified.
+
+Then, in the :guilabel:`Sales` section, in the :guilabel:`Visible at` field, click the drop-down
+menu, and select either: :guilabel:`Quotation`, :guilabel:`Confirmed order`, or :guilabel:`Inside
+quote`.
+
+- :guilabel:`Quotation`: the document is sent to (and accessible by) customers at any time.
+
+- :guilabel:`Confirmed order`: the document is sent to customers upon the confirmation of an order.
+  This is best for user manuals and other supplemental documents.
+
+- :guilabel:`Inside quote`: the document is included in the PDF of the quotation, between the header
+  pages and the :guilabel:`Pricing` section of the quote.
+
+.. example::
+   When the :guilabel:`Inside quote` option for the :guilabel:`Visible at` field is chosen, and the
+   custom PDF file, `Sample Builder.pdf` is uploaded, the PDF is visible on the quotation the in the
+   *customer portal* under the :guilabel:`Documents` field.
+
+    .. image:: pdf_quote_builder/pdf-inside-quote-sample.png
+       :align: center
+       :alt: Sample of an uploaded pdf with the inside quote option chosen in Odoo Sales.
+
+Lastly, in the :guilabel:`E-Commerce` section, decide whether or not to :guilabel:`Show on product
+page` on the front-end (in the online store).
+
+.. example::
+   When the :guilabel:`Show on product page` option is enabled, a link to the uploaded document,
+   `Sample Builder.pdf`, appears on the product's page, located on the frontend in the online store.
+
+   It appears beneath a :guilabel:`Documents` heading, with a link showcasing the name of the
+   uploaded document.
+
+    .. image:: pdf_quote_builder/show-product-page.png
+       :align: center
+       :alt: Showing a link to an uploaded document on a product page using Odoo Sales.
+
+PDF quote
+=========
+
+Once a quote with a pre-configured PDF has been confirmed, Odoo provides the option to print the
+confirmed quote to check for errors, or to keep for records.
+
+To print the PDF quote, navigate to the confirmed quote, and click the :guilabel:`⚙️ (gear)` icon to
+reveal a drop-down menu. From this drop-down menu, select :guilabel:`Print`, then select
+:guilabel:`PDF Quote`.
+
+.. image:: pdf_quote_builder/drop-down-print-pdf.png
+   :align: center
+   :alt: Print pdf quote option on drop-down menu located on confirmed sales order in Odoo Sales.
+
+Doing so instantly downloads the PDF quote. When opened, the PDF quote, along with the configured
+product PDF that was set to be visible inside the quote, can be viewed and printed.
```

Comment:
I agree it would be great to include an example of a finished quote with an attached PDF here at the end.